int32 id
---
int32 id
geometry_msgs/Point HomePos
int32 PosNum
tsu200/Pos[] PosList
